src/server/pingserver/admin/process.c

```diff
@@ -1,7 +1,7 @@
 #include "process.h"
 
-#include <protocol/admin/admin_include.h>
-#include <util/procinfo.h>
+#include "../../../protocol/admin/admin_include.h"
```

Comment:
All these are fairly ugly.
You can drop the leading `../../../` and it would still work. In this case it will just fall back to search in the paths specified by `-I`, and we have added the top-level `src/` to the include directories.